packages/pg-v5/lib/psql.js

```diff
@@ -49,9 +49,22 @@ function execPsqlWithFile (file, dbEnv) {
 function psqlInteractive (dbEnv, prompt) {
   const {spawn} = require('child_process')
   return new Promise((resolve, reject) => {
-    let psql = spawn('psql',
-      ['--set', `PROMPT1=${prompt}`, '--set', `PROMPT2=${prompt}`],
-      {env: dbEnv, stdio: 'inherit'})
+    let psql_args = ['--set', `PROMPT1=${prompt}`, '--set', `PROMPT2=${prompt}`, ]
```

Comment:
@mage2k please change to camel case style `psqlArgs`

Reply:
Will do.